### ISSUE TITLE
'run.py' line44

### ISSUE THREAD

**Issue by jiminbot20** [lukewys/dcase_2020_T6#3]
This is line 44 in _run.py_
**caption_model_path ='models/augmentation+smoothing+pretrain_cnn+freeze_cnn/20.pt'**
Then this error occurs
_FileNotFoundError: [Errno 2] No such file or directory: 'models/augmentation+smoothing+pretrain_cnn+freeze_cnn/20.pt'_
![image](https://user-images.githubusercontent.com/63258184/93317729-c5e9f780-f848-11ea-965a-89b163fda73b.png)

![image](https://user-images.githubusercontent.com/63258184/93317653-b2d72780-f848-11ea-9283-811338d91cf7.png)

there is no '20.pt' then '18.pt'. so is it fine to change line 44 to code below?
**caption_model_path ='models/18.pt'**

**Comment by lukewys** [comment]
Did you train the full 30 epochs of "augmentation+smoothing+pretrain_cnn+freeze_cnn"?

So this is actually searching for the model that saved in the previous training. You should train the full 30 epoch, and you should find a 20.pt in the folder.

For your last question, it is ok to load models/18.pt, which gives you the model of the 18th epoch of the aforementioned training.

**Comment by jiminbot20** [comment]
![image](https://user-images.githubusercontent.com/63258184/93326239-752bcc00-f853-11ea-9763-71fb2c12d067.png)
 
I think I'm not training 20 epochs. Where can I fix epochs to train in 'run.py' or 'train.py'?

this is 'run.py' screenshot below. training_epochs is set to 30 but 1 epoch is running as you see in above screenshot. Could you help me?
![image](https://user-images.githubusercontent.com/63258184/93326523-e4a1bb80-f853-11ea-8990-0fe2349547e8.png)

**Comment by jiminbot20** [comment]
I fixed the problem. It was my mistake. after running 1 epoch, there was error on evaluating SPICE metric(I didn't run .sh file in coco_caption). Thank you

**Comment by lukewys** [comment]
Hi @jiminbot20, no problem! Glad you succeed :).
